edxval/models.py

```diff
@@ -518,15 +547,10 @@ def create_or_update(cls, video, language_code, metadata, file_data=None):
 
         transcript_name = metadata.get('file_name')
 
+        video_transcript.save()
+
```

Comment:
Why this save here?

Reply:
That's a line I changed and it went unnoticed by me, sorry about that.